tests/test_context.py

```diff
@@ -385,7 +386,21 @@ def test_component_excludes_variable_with_isolated_context_from_settings(
 
 
 class OuterContextPropertyTests(SimpleTestCase):
-    def test_outer_context_property_with_component(self):
+    @override_settings(
+        COMPONENTS={"context_behavior": "global"},
+    )
+    def test_outer_context_property_with_component_global(self):
+        template = Template(
+            "{% load component_tags %}{% component_dependencies %}"
+            "{% component 'outer_context_component' only %}{% endcomponent %}"
+        )
+        rendered = template.render(Context({"variable": "outer_value"})).strip()
+        self.assertIn("outer_value", rendered, rendered)
+
+    @override_settings(
+        COMPONENTS={"context_behavior": "isolated"},
+    )
+    def test_outer_context_property_with_component_isolated(self):
```

Comment:
Added test to cover both options for `context_behavior`

Reply:
If a `context_behavior="isolated"` wouldn't that mean that in this case it shouldn't display `outer_value`?

Reply:
Good catch! So in this particular test, the outer context does not leak into `outer_context_component` component. But the component itself manually [accesses the outer context via `self.outer_context` in `get_context_data`](https://github.com/EmilStenstrom/django-components/pull/437/files/4ffccdf5dd9ab7b0c5d34c29928ed45efe55db65#diff-0739130667ad94af9f1ac9b3ab331c42b6e4c5b0fe7bc42a3dbb47f900b7cfcaR69).

My intuition says that the `outer_context` should be available even if the templates are isolated, since `outer_context` could have also other uses. But do correct me if that's not how to feature was intented to work.

Reply:
Ah you’re right! Makes sense. 

This is very thoroughly tested! 👍